flowcraft/generator/templates/mlst.nf

```diff
@@ -47,17 +47,17 @@ process mlst_{{ pid }} {
 
 process compile_mlst_{{ pid }} {
```

Comment:
If for some reason, this process in kept, then the name does not make sense anymore, as it is not compiling stuff.

Reply:
If the results are being produced for each sample, then the whole `compile_mlst` process is not needed anymore. You could just remove this process and add the publishDir of the appropriate output in the mlst process correct?